staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/metrics.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kmsv2
```

Comment:
Can we just use the old metrics so that any consumers don't need to look in a new location?

Reply:
Good suggestion! This being an implementation change that's not really visible to users, reusing the metrics makes more sense. I've moved metrics to it's own pkg and reused it for v1 and v2. LMK what you think?